include/dsn/dist/replication/replication.types.h

```diff
@@ -469,6 +469,14 @@ namespace dsn { namespace replication {
     {
         int32_t app_id;
         int32_t pidx;
+        inline bool operator < (const global_partition_id& r) const
```

Comment:
"users should include instead of the internal"  --> some words are hidden by markdown.